README.md

```diff
@@ -1,90 +1,50 @@
 # ewasm testnet coordination / documentation repo
 
-## Aleth (ex cpp-ethereum)
+## Differences from mainnet
 
-To build Aleth with the recent ewasm changes use [ewasm](https://github.com/ethereum/aleth/tree/ewasm).
-
-`aleth`, `alethvm` and `testeth` contain options to run them with [Hera ewasm VM](https://github.com/ewasm/hera):
-
-- `--vm hera` enables Hera only,
-- `--evmc fallback=true` enables fallback to EVM 1.0 Interpreter when EVM bytecode is detected (off by default)
-
-### Test net differences from main net
-
-Supports executing EVM 1.0 (Byzantium) **and** ewasm bytecode. The chain id is set to 0x42 (66).
+The Ewasm testnet supports executing EVM 1.0 (Byzantium) bytecode **and** ewasm bytecode. The chain id is set to 0x42 (66).
 
 There are two differences:
```

Comment:
@axic could you or someone else please insert some of the rationale for these changes?

Reply:
To differentiate ourselves from mainnet?

Reply:
To prevent replay attacks from other chains.

Reply:
I meant, what's the rationale behind the following two lines?

- code size limit introduced by Spurious Dragon has been lifted and there is no upper limit
- zero bytes in contract bytecode are not subsidised anymore during deployment (they cost the same as non-zero bytes)

Reply:
Ping @jwasinger @ewasm/ewasm-team can anyone clarify this a bit? Thanks

Reply:
> zero bytes in contract bytecode are not subsidised anymore during deployment (they cost the same as non-zero bytes)

I don't understand the reasoning behind this.  The reason of subsidizing zero bytes is because they can be compressed.

> code size limit introduced by Spurious Dragon has been lifted and there is no upper limit

wasm bytecode is a lot larger than EVM and we lifted the limit so people can deploy more things.